horace_core/symop/@Symop/Symop.m

```diff
@@ -237,7 +237,17 @@ function disp(obj)
                         warning('HORACE:deprecated_function',...
                             'spher_proj is deprecated. Use sphere_proj instead')
 
-                    end
+                    %% TODO non-aligned ez/ey not supported
+                    % ez_new = obj.R * proj.ez(:);
+                    % ey_new = obj.R * proj.ey(:);
+
+                    %                 offset_new = proj.offset(:);
+                    %                 offset_new(1:3) = obj.transform_vec(offset_new(1:3));
+                    %
+                    %                 proj.offset = offset_new;
+                case 'spher_proj'
```

Comment:
this looks like  a clash or duplicate with line 236

Reply:
Note the lack of an `e`

Reply:
Nope, you're right. Merge/rebase/whatever error

Reply:
That entire thing shouldn't even be there, the deprecation would be caught when the objects were instantiated. Removed